### DISCUSSION TITLE
Report this page when asked at the time of support... ==> TypeError: Cannot read properties of undefined (reading 'id')     at handleRequest (worker.js:1337:26)

### DISCUSSION THREAD

**Discussion by erickyun** [anasty17/mirror-leech-telegram-bot#1563] [Q&A]
Report this page when asked at the time of support... ==> TypeError: Cannot read properties of undefined (reading 'id')
    at handleRequest (worker.js:1337:26)

Hi,

I'm using the lastest https://gitlab.com/GoogleDriveIndex/Google-Drive-Index/-/blob/master/src/worker.js?ref_type=heads
index from Bhadoo index but yet I'm receiving this error?
What am I doing wrong I would appreciate if someone could please help me?

**Comment by anasty17** [comment]
Ask in bhadoo support group. I can't directly know the issue
